pending/potplayer.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    BleachBit
+    Copyright (C) 2008-2019 Andrew Ziem
+    https://www.bleachbit.org
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+    @app Daum PotPlayer
+    @url http://potplayer.daum.net/
+    @cleanerversion v0.2.5
+    @os Windows
+    @cleanerdate 2019-03-18
+    @cleanerby https://github.com/Tobias-B-Besemer
+    @tested ok v???, Windows 7
+    @testeddate 2019-02-25 (... with an old version of the cleaner!)
+    @testedby https://github.com/Tobias-B-Besemer
+    @note This cleaner is "unstable"! Please improve it before you use it! (Or do whatever you like! ;-))
+    @note That we don't need so much translation, we change some things... Tobias.
+
+-->
+<cleaner id="potplayer" os="windows">
+  <label>PotPlayer</label>
+  <description>Media player</description>
+  <!--
+  That we don't need so much translation, we change some things... Tobias.
+  Was:
+  <option id="rememberfiles">
+    <label>Remember Files</label>
+    <description>Delete the data</description>
+  Is now: 
+  -->
+  <option id="mru">
+    <label>Most recently used</label>
+    <description>Delete the list of recently used documents</description>
+    <action command="winreg" path="HKCU\Software\Daum\PotPlayerMini\RememberFiles\"/>
+    <action command="winreg" path="HKCU\Software\Daum\PotPlayerMini64\RememberFiles\"/>
+  </option>
+  <option id="logs">
+    <label>Logs</label>
+    <description>Delete the logs</description>
+    <action command="delete" search="walk.files" path="%AppData%\Daum\PotPlayer\Log\*"/>
```

Comment:
Do you mean every directory under `%AppData%\Daum\PotPlayer\Log\`? If yes, then `walk.files` is right, but if you want every file then you want either

 `search="glob"` (which will not delete recursively)
OR
remove the wildcard  (which will delete recursively)

Reply:
OK, fixed it!